doc/berrymill.8.md

```diff
@@ -0,0 +1,280 @@
+**\--box-debug** to the boxbuild.
+
+-a ARCH, \--arch ARCH
+
+: Sets target architecture e.g., aarch64.
+
+-c CONFIG, \--config CONFIG
+
+: Specify a configuration file other than default one. Expects a configuration
+file as argument. Read more about the structure of configuration files in
+section [FILES](#files).
+
+-i IMAGE, \--image IMAGE
+
+: Path to the image description, this parameter is required.
+For more information on kiwi consult **kiwi**(8).
+
+-p PROFILE, \--profile PROFILE
+
+: Select a profile from the image description. If more than one is provided
+in the image description and this parameter is not provided the build will
+fail.
+
+\--clean
+
+: When \--clean is passed **berrymill** will cleanup previous build results like
+the target directory.
+
+When **berrymill** is executed with the option *prepare*  to prepare the sysroot
+the following options are additionally available.
+
+\--root ROOT
+
+: Specifies path where the root system shall be created. This is passed along
+to **kiwi::system::prepare**(8). This parameter is required for *prepare*.
+
+\--allow-existing-root
+
+: Allow to re-use an existing image root directory. This parameter is passed
+along to **kiwi::system::prepare**(8).
+
+When **berrymill** is executed with the option *build*  to build the image the
+following options are additionally available.
+
+\--box-memory BOX\_MEMORY
+
+: Specify amount of memory to be used by qemu for the boxbuild. The default is
+*8G*. This parameter is passed to **kiwi::system::boxbuild**(8).
+
+\--cpu CPU
+
+: Specify CPU type to be used by qemu for the boxbuild. Per default the host
+CPU is used. This parameter is passed to **kiwi::system::boxbuild**(8).
+This option has no effect in combination with \--cross, as \--cross overwrites
+this setting to an appropriate one.
+
+\--cross
+
+: Select \--cross to cross build an image on a x86\_64 host to an aarch64
+target.
+
+-l, \--local
+
+: If -l, \--local is selected a local image build on the current architecture
+is performed instead of a boxbuild. This will require sudo rights and an
+installation of the KIWI tool chain.
+
+\--target-dir TARGET\_DIR
+
+: Specifiy the directory where the results of the build shall be placed by
+**berrymill**. This parameter is required and gets passed along to
+**kiwi**(8).
+
+\--no-accel
+
+: Disables the KVM acceleration for boxbuild.
+
+\--ignore-nested
+
+: Ignore warning that nested virtualization is not enabled.
+
+Exit status
+-----------
+
+**berrymill** returns 0 on success else 1.
+
+FILES
+=====
+
+* */etc/berrymill/berrymill.conf*
+
+This file defines the repositories that **berrymill** uses to build the image.
+An example structure is given in the following.
+
+```
+obs-api: <api_url>
+
+use-global-repos: false
+boxed_plugin_conf: <path_to_boxplugin_conf>
+repos:
+  release:
+    <architecture_1>
+      Ubuntu-Jammy:
+        url:  <repo_url>
+        type: <repo_type>
+        name: <name>
+        components: main,universe,...
+
```

Comment:
Please mention this somehow:
https://github.com/isbm/berrymill/blob/main/config/berrymill.conf.example#L33